src/components/styled/Header.styled.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+export const StyledHeader = styled.div`
```

Comment:
Instead of using a `div` you can also use a `header` element. That would be more semantic.